### DISCUSSION TITLE
Derive attribute on structs generated by macro select! , include!

### DISCUSSION THREAD

**Discussion by goni098** [Brendonovich/prisma-client-rust#365] [Q&A]
For example 
```
post::select!(post_only_title {
    title
})

// Generated type 
pub mod post_only_title {
    pub struct Data {
        title: String
    }
}
```
Is there any way to derive some trait to post_only_title::Data struct 
Such as `#[derive(ToSchema)], #[schema(example = json!({"title": "bob the cat"}))]`

**Comment by Brendonovich** [comment]
This isn't possible and idk if it should be possible, since this wouldn't be super useful without also being able to add derives to regular `Data` structs
